Source/SwiftLintFramework/Rules/Lint/UnusedSetterValueRule.swift

```diff
@@ -41,6 +41,14 @@ public struct UnusedSetterValueRule: ConfigurationProviderRule, AutomaticTestabl
                     Persister.shared.aValue = value
                 }
             }
+            """),
+            Example("""
+            override var aValue: String {
+             get {
+                 return Persister.shared.aValue
+             }
+             set() { }
```

Comment:
These examples get automatically run as tests, right?

This example fails for me, on v0.46.2

```shell
echo 'private var abc: Int { get { 123 } set() { } }' | swiftlint lint --no-cache --use-stdin --enable-all-rules
```

```
<nopath>:1:36: warning: Unused Setter Value Violation: Setter value is not used. (unused_setter_value)
Done linting! Found 1 violation, 0 serious in 1 file.
```

Can you reproduce?

Reply:
Yes these are run as tests, please open a new issue with full steps to reproduce, what you're seeing and what you're expecting.

Reply:
Opened https://github.com/realm/SwiftLint/issues/3863 :)